### PR TITLE
fix: Adjust profile screen padding, typography and add divider

### PR DESCRIPTION
**Pull request by dexterca**
This PR is trying to better match styles found in Figma file. This is my first commit to a React Native project so I thought I'd start small.

## Changes
- Adjust font sizes
- Add fields container padding
- Adjust spacing between fields
- Add divider and adjust top margin

## Screenshots
BEFORE | AFTER
-- | --
Simulator Screenshot - iPhone 15 Pro - 2023-11-10 at 10 47 29 | Simulator Screenshot - iPhone 15 Pro - 2023-11-10 at 10 46 04

## Related PR
https://github.com/lifeomic/fountain-life-member-app/pull/152

### CODE REVIEW

**Review by jkdowdle — APPROVED** [review]
This looks great! nice improvement, I think the only thing would be to make sure this is desired to go out to other use cases too for Branded Mobile Apps

File: src/screens/ProfileScreen.tsx [hunk 1]
```diff
@@ -127,12 +128,12 @@ const Field = ({ label, value }: FieldProps) => {
...
-        {label}
-      </Text>
-      <Text variant="bodyLarge" style={styles.fieldValueText}>
-        {value}
-      </Text>
+      <Text style={styles.fieldLabelText}>{label}</Text>
+      <Text style={styles.fieldValueText}>{value}</Text>
+      <Divider
+        style={styles.divider}
+        theme={{ colors: { outlineVariant: '#727983' } }}
```
Comment (jkdowdle) [inline src/screens/ProfileScreen.tsx:135]: Could consider pulling this theme object out of the component

Comment (aecorredor) [reply src/screens/ProfileScreen.tsx:135]: I think the divider should take the color directly from the theme that comes from the root theme provider.

**Review by aecorredor — COMMENTED** [review]
I'd +1 Josh's comment about not providing a theme object explicitly at that level. Like for other components, there should be a way for the divider to take the global theme color.

**Review by aecorredor — APPROVED** [review]

**Pull request merged**